packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/tools/parseProposals.mjs

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+import fs from 'fs';
+
+const Fail = (strs, _things) => {
+  throw new Error(strs.join(' ')); // TODO: include things
+};
+
+/**
+ * Parse output of `agoric run proposal-builder.js`
```

Comment:
An alternative is to teach `agoric run` to produce parseable output. This is the 2nd time I didn't opt to do that.

Reply:
Maybe someday. For testing purposes I think we'd just add a parse-smart version to `upgradeHelper.js` once that's available